sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/MetricStorage.java

```diff
@@ -20,6 +20,9 @@
  */
 public interface MetricStorage {
 
+  /** The max number of metric accumulations for a particular {@link MetricStorage}. */
+  Integer MAX_ACCUMULATIONS = 2000;
```

Comment:
This should be private somewhere not public for end users as when added to an interface.

Reply:
Also why is this `Integer` ?

Reply:
> Also why is this Integer

Ah yes, can be `int`.